src/main/java/edu/SlowestKeySolution.java

```diff
@@ -1,9 +1,78 @@
 package edu;
 
+import java.util.Arrays;
+
 public class SlowestKeySolution {
     public static final char INVALID = '-';
 
+
+//    String keysPressed = "asdagrehhg";
+//    int[] releaseTimes = new int[] { 1, 3, 4, 5, 8, 9, 10, 13, 14, 17};
+//        SlowestKeySolution.solve(releaseTimes, keysPressed);
+
     public static char solve(int[] releaseTimes, String keysPressed) {
-        return INVALID;
-    }
-}
+        if (keysPressed == null || keysPressed.equals("")) {
+            return INVALID;
+        }  else if (releaseTimes == null) {
+            return INVALID;
+        } else if (!(releaseTimes.length == keysPressed.length())) {
+            return INVALID;
+        }
+        char[] ch = new char[keysPressed.length()];
+                for (int i = 0; i < keysPressed.length(); i++) {
+                    ch[i] = keysPressed.charAt(i);
+                }
+                System.out.println(ch);
+        int  b = (int) ch[0];
+        char tempChar2 = '0';
+        System.out.println(b);
+                int[] durations = new int[releaseTimes.length];
+                int tempInt = 0;
+                char tempChar = 'a';
+                durations[0] = releaseTimes[0];
+                System.out.println(Arrays.toString(durations));
+                for (int i = 1; i < releaseTimes.length; i++) {
+                    int releaseTimesDuration = releaseTimes[i] - releaseTimes[i - 1];
```

Comment:
There is no need to initiate a variable every loop iteration. The result can be assigned directly to durations [i].